### PR TITLE
[nnc] Updated generic error message with info about turning off the fuser

### PR DESCRIPTION
**Pull request by navahgar**
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #64329
* #64328
* #64319
* #64318
* #64317
* __->__ #64316

Differential Revision: [D30683942](https://our.internmc.facebook.com/intern/diff/D30683942)

### CODE REVIEW

**Comment by navahgar** [comment]
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30683942).

File: torch/csrc/jit/tensorexpr/kernel.cpp [hunk 1]
```diff
@@ -67,10 +67,16 @@ namespace jit {
 namespace tensorexpr {
 
 std::string buildErrorMessage(const std::string& s) {
-  // TODO: Update this generic error message to include details regarding
-  // turning off the fuser.
-  static const std::string generic_error_message = "";
-  return s + " " + generic_error_message;
+  static const std::string generic_error_message =
+      "This error occured in the fuser. You can turn off the fuser by setting "
+      "torch._C._jit_override_can_fuse_on_cpu to false";
```
Comment (bertmaher) [inline torch/csrc/jit/tensorexpr/kernel.cpp:72]: Maybe "You can turn off the fuser with torch._C._jit_override_can_fuse_on_cpu(False)"
Although it's kind of annoying if it's the GPU fuser but 🤷

**Review by bertmaher — APPROVED** [review]

**Comment by navahgar** [comment]
@navahgar has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D30683942).

File: torch/csrc/jit/tensorexpr/kernel.cpp (hunk 1 above)
Comment (navahgar) [reply torch/csrc/jit/tensorexpr/kernel.cpp]: Updated that. 

Good point regarding GPU fuser. Should we provide one flag to turn both off?

**Pull request closed without merging**